easybuild/easyconfigs/m/matplotlib/matplotlib-2.0.0-CrayGNU-17.08-Python-2.7.13.eb

```diff
@@ -0,0 +1,48 @@
+# Contributed by Theofilos Manitaras (cscs)
```

Comment:
You should use https://github.com/eth-cscs/production/blob/master/easybuild/easyconfigs/p/PyExtensions/PyExtensions-2.7-CrayGNU-17.08.eb#L28